setup.py

```diff
@@ -38,7 +37,7 @@
 
 setup(
     name="titiler",
-    version="0.1.0a8",
+    version="0.1.0a9",
```

Comment:
🤦 I reset my local branch one commit too early.
FYI: I'm going going to change this, I think it's fine!